input/fsh/SD_TumorSize.fsh

```diff
@@ -15,7 +15,6 @@ Description:  "Records the dimensions of a tumor"
 * subject 1..1 MS
```

Comment:
Throughout, ^short should be a name or short description and ^definition should be a description. You don't need to put conformance expectations in the definition, since the `only` statement clearly sets the data type.